app/models/scan_item.rb

```diff
@@ -54,15 +54,15 @@ def self.seed
 
   def self.preload_default_profile
     # Create sample VM scan profiles
-    vm_profile = ScanItemSet.find_or_initialize_by_name(SAMPLE_VM_PROFILE)
+    vm_profile = ScanItemSet.find_or_initialize_by(:name => SAMPLE_VM_PROFILE[:name])
```

Comment:
Alternately, since the next line calls update_attributes and saves to the database anyway, you can do:

`vm_profile = ScanItemSet.create_with(SAMPLE_VM_PROFILE).find_or_create_by(SAMPLE_VM_PROFILE.slice(:name))`